frontend/test/pytest/test_transform.py

```diff
@@ -392,7 +392,9 @@ def qfunc():
         assert np.allclose(expected, observed)
         _, expected_shape = jax.tree_util.tree_flatten(expected)
         _, observed_shape = jax.tree_util.tree_flatten(observed)
-        assert expected_shape == observed_shape
+        assert expected_shape.num_leaves == observed_shape.num_leaves
+        # TODO: See https://github.com/PennyLaneAI/catalyst/issues/1099
+        # assert expected_shape == observed_shape
```

Comment:
In the same file there's the same issue in `test_expansion_qnode_no_cache` I think. Maybe we can add the issue link to there as well?